security/advancedtls/advancedtls_test.go

```diff
@@ -679,3 +681,85 @@ func TestOptionsConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCertificateSNI(t *testing.T) {
+	const (
```

Comment:
I think it's quite uncommon to have a const inside a function ... directly assign a 0 value doesn't work? You can also make it a local variable and enforce its type.

Reply:
Sure, I'll change it to a local variable. Just thought it's more clear and consistent to assign a variable name that matched with other files.